kyo-zio/shared/src/main/scala/kyo/zios.scala

```diff
@@ -0,0 +1,68 @@
+package kyo
+
+import ZIOs.internal.*
+import core.*
+import core.internal.*
+import kyo.fibersInternal.*
+import scala.util.control.NonFatal
+import zio.Task
+import zio.ZIO
+
+opaque type ZIOs <: Fibers = Tasks & Fibers
+
+object ZIOs:
+
+    def get[E >: Nothing: Tag, A](v: ZIO[Any, E, A]): A < (Aborts[E] & ZIOs) =
+        val task = v.fold[A < Aborts[E]](e => Aborts.fail(e), a => a)
+        Tasks.suspend[A < Aborts[E], A, Aborts[E]](task, identity)
+
+    def get[A](v: ZIO[Any, Nothing, A]): A < ZIOs =
+        Tasks.suspend[A](v)
+
+    inline def get[R: zio.Tag, A](v: ZIO[R, ?, A])(using tag: Tag[Envs[R]]): A < (Envs[R] & ZIOs) =
```

Comment:
I think this function needs the `E`. Even if we later plan to add overloads for `Nothing` case, it's better for compatibility to keep the `E`

```
inline def get[R: zio.Tag, E, A](v: ZIO[R, E, A])(using tag: Tag[Envs[R]]): A < (Envs[R] & ZIOs) =
```

Reply:
done